ReadMe_cn.md

```diff
@@ -0,0 +1,985 @@
+# 【Zig】乐器数字接口-Midi
```

Comment:
While I respect the effort in writing these docs, I cannot accept a readme in a language I don't understand. It is not really something I can maintain (even though I don't really work on this anymore) and it could anything. I could run it through some automated translation to get an idea of the meaning but that is still quite imperfect.

Reply:
> While I respect the effort in writing these docs, I cannot accept a readme in a language I don't understand. It is not really something I can maintain (even though I don't really work on this anymore) and it could contain anything. I could run it through some automated translation to get an idea of the meaning but that is still quite imperfect.

I would rewrite it in English. Don't worry.